stellarphot/io/tess.py

```diff
@@ -18,13 +18,61 @@
 __all__ = ["TessSubmission", "TOI", "TessTargetFile"]
 
 # Makes me want to vomit, but....
-DEFAULT_TABLE_LOCATION = "who.the.fuck.knows"
+DEFAULT_TABLE_LOCATION = "who.the.heck.knows"
 TOI_TABLE_URL = "https://exofop.ipac.caltech.edu/tess/download_toi.php?output=csv"
 TIC_regex = re.compile(r"[tT][iI][cC][^\d]?(?P<star>\d+)(?P<planet>\.\d\d)?")
 
 
 @dataclass
 class TessSubmission:
+    """ A data class to represent TESS submissions.
```

Comment:
```suggestion
    """
    A data class to represent TESS submissions.
```
Definitely don't need the "data" part -- it is essentially just a shorthand way to make python handle a bunch of the drudgery for you.

Reply:
Addressed in a manual edit.